coreMacros/src/main/scala-3/me/shadaj/scalapy/py/Facades.scala

```diff
@@ -0,0 +1,171 @@
+package me.shadaj.scalapy.py
+
+import scala.quoted.*
+import scala.annotation.StaticAnnotation
+class native extends StaticAnnotation
+
+class FacadeCreator[T]
```

Comment:
Do we need this and the `Any` declaration below?

Reply:
Otherwise we will get `[info]   java.lang.NoClassDefFoundError: Could not initialize class me.shadaj.scalapy.py.package$` error in the tests. Also confused why it happens.